pkgs/applications/emulators/craftos-pc/default.nix

```
@@ -14,29 +14,30 @@
 }:
 
 let
+  version = "2.7.2";
   craftos2-lua = fetchFromGitHub {
     owner = "MCJack123";
     repo = "craftos2-lua";
-    rev = "v2.6.6";
-    sha256 = "cCXH1GTRqJQ57/6sWIxik366YBx/ii3nzQwx4YpEh1w=";
+    rev = "v${version}";
+    sha256 = "sha256-g0cYbAKAK8SEdjTID7FFus9rncfSPYEUfuQaatu9F0s=";
   };
   craftos2-rom = fetchFromGitHub {
     owner = "McJack123";
     repo = "craftos2-rom";
-    rev = "v2.6.6";
-    sha256 = "VzIqvf83k121DxuH5zgZfFS9smipDonyqqhVgj2kgYw=";
+    rev = "v${version}";
+    sha256 = "sha256-UaachE3lonHrNaOPrM3oAIQ8Kq0NuryytS6/sypF6fw=";
   };
 in
 
 stdenv.mkDerivation rec {
   pname = "craftos-pc";
-  version = "2.6.6";
+  version = version;
```

```suggestion
  inherit version;
```

Otherwise you get an infinite recursion due to the `rec`.

---

Fixed.